src/main/java/appeng/tile/misc/CondenserItemInventory.java

```diff
@@ -107,4 +110,14 @@ public boolean validForPass( final int i )
 	{
 		return i == 2;
 	}
+
+	@Override
+	public void addListener( IMEMonitorHandlerReceiver<IAEItemStack> l, Object verificationToken )
+	{
+	}
+
+	@Override
+	public void removeListener( IMEMonitorHandlerReceiver<IAEItemStack> l )
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)